src/popup/settings.js

```diff
@@ -195,12 +196,12 @@ export default class Settings {
   }
 
   /**
-   * Send a message to the tab to set the autoSave value.
+   * Send a message to the tab to set an auto setting value.
```

Comment:
Function to enable sending messages to the inject script for tasks that may be run automatically.